testcases/PowerNVDump.py

```diff
@@ -166,31 +170,39 @@ def verify_dump_dt_node(self, boot_type=BootType.NORMAL):
             self.c.run_command("lsprop %s/mpipl-boot" % BMC_CONST.OPAL_DUMP_NODE)
 
     def verify_dump_file(self, boot_type=BootType.NORMAL, dump_place="local"):
+        dump_path = "/var/crash"
         if dump_place == "local":
             crash_content_after = self.c.run_command(
-                "ls -l /var/crash | grep '^d'| awk '{print $9}'")
+                "ls -l %s | grep '^d'| awk '{print $9}'" % dump_path)
         if dump_place == "net":
             crash_content_after = self.c.run_command(
-                "ssh root@%s \"ls -l /var/crash | grep '^d'\" | awk '{print $9}'" % self.server_ip)
+                "ssh root@%s \"ls -l %s | grep '^d'\" | awk '{print $9}'" % (self.server_ip, dump_path))
+        if dump_place == "san":
+            dump_path = "/mnt/var/crash"
+            crash_content_after = self.c.run_command(
+                "ls -l %s | grep '^d'| awk '{print $9}'" % dump_path)
         self.crash_content = list(
             set(crash_content_after) - set(self.crash_content))
         if len(self.crash_content):
             if dump_place == "net":
                 self.c.run_command('scp -r root@%s://var/crash/%s /var/crash/' %
                                   (self.server_ip, self.crash_content[0]), timeout=600)
             if self.distro == "ubuntu":
-                self.c.run_command("ls /var/crash/%s/dump*" %
-                                   self.crash_content[0])
+                self.c.run_command("ls %s/%s/dump*" %
+                                   (dump_path, self.crash_content[0]))
+            elif dump_place == "san":
+                self.c.run_command("ls %s/%s/vmcore*" %
+                                   (dump_path, self.crash_content[0]))
             else:
-                self.c.run_command("ls /var/crash/%s/vmcore*" %
-                                   self.crash_content[0])
+                self.c.run_command("ls %s/%s/vmcore*" %
+                                   (dump_path, self.crash_content[0]))
```

Comment:
@kumudag Some how the logic in the above code is confusing, if is san or non-san only the dump_path is going to change. In our case already we have dump_path, in my opinion if is net copy the dump to the /var/crash, that we have are doing it. Next, if it is ubuntu use dumpfile name as "dump" else use it as "vmcore". That should solve the purpose, why the "san" check again here.